### PR TITLE
Boot info

### PR DESCRIPTION
**Pull request by Taylorr82**

### CODE REVIEW

File: source/bl_eeprom.c [hunk 1]
```diff
@@ -10,6 +10,16 @@
 
 unsigned short crc16();
 
+void sw_reset(SW_RESET_REASON reason) {
```
Comment (arrooney) [inline source/bl_eeprom.c:13]: Seems like a bit of an odd place to have this function, but maybe it's worth the simplicity

**Review by arrooney — APPROVED** [review]
How will the boot reason be logged in the error condition? Will that be a todo item?

File: source/bl_eeprom.c (hunk 1 above)
Comment (Taylorr82) [reply source/bl_eeprom.c:13]: I put it here since this file gets downloaded to the obc software and it seemed good to have it accessible there

**Comment by Taylorr82** [comment]
> How will the boot reason be logged in the error condition? Will that be a todo item?

What do you mean?

**Review by wagnerd97 — APPROVED** [review]

**Pull request merged**